doc/changes/latest.rst

```diff
@@ -2,62 +2,126 @@
 
 .. include:: names.rst
 
-0.11.0.dev
+0.10.3.dev
 ==========
 
 HIGHLIGHTS
 ----------
 
+.. warning::
+
+ | **Minimum supported versions of packages have been bumped up:**
+ | - Numpy -- v1.19.0
+ | - SciPy -- v1.8.0
+ | - Scikit-learn -- v1.0.0
+ | - Nibabel -- v4.0.0
+ | - Pandas -- v1.1.5
+ | - Joblib -- v1.0.0
+
+
+- :bdg-success:`API` Allow passing arguments to :func:`~nilearn.glm.first_level.first_level_from_bids` to build first level models that include specific set of confounds by relying on the strategies from :func:`~nilearn.interfaces.fmriprep.load_confounds` (:gh:`4103` by `Rémi Gau`_).
+
+- :bdg-success:`API` Support passing t and F contrasts to :func:`~nilearn.glm.compute_contrast` that that have fewer columns than the number of estimated parameters. Remaining columns are padded with zero (:gh:`4067` by `Rémi Gau`_).
+
+- :bdg-success:`API` :class:`~maskers.NiftiSpheresMasker` now has ``generate_report`` method (:gh:`3102` by `Yasmin Mzayek`_ and `Nicolas Gensollen`_).
+
+- :bdg-dark:`Code` Update the ``CompCor`` strategy in :func:`~interfaces.fmriprep.load_confounds` and :func:`~interfaces.fmriprep.load_confounds_strategy` to support ``fmriprep`` 21.x series and above. (:gh:`3285` by `Hao-Ting Wang`_).
+
+- :bdg-info:`Plotting` Allow setting ``vmin`` in :func:`~nilearn.plotting.plot_glass_brain` and :func:`~nilearn.plotting.plot_stat_map` (:gh:`3993` by `Michelle Wang`_).
+
+- :bdg-info:`Plotting` When plotting thresholded statistical maps with a colorbar, the threshold value(s) will now be displayed as tick labels on the colorbar (:gh:`#2833` by `Nicolas Gensollen`_).
+
+- :bdg-primary:`Doc` Combine GLM examples ``plot_fixed_effect`` and ``plot_fiac_analysis`` into a a single example ``plot_two_runs_model`` (:gh:`3191` by `Taylor Salo`_ and `Rémi Gau`_).
```

Comment:
those are the things that stood out to me as worth highlighting: have a look at the content to the rest of the changelog and let me know if something else should go in.